common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -26,6 +26,7 @@
 #include <Eigen/Geometry>
 
 #include <autoware_auto_planning_msgs/msg/path.hpp>
+#include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
```

Comment:
Since PathWithLaneId is a TIER IV's extension message, @yukkysaito intentionally avoided adding this to `tier4_autoware_utils`.

Reply:
Right. 
I had avoided it when it was .iv's common, but now it is tier4_planning_utils, so it might be good. 
but It will be put into common.

If it is possible, it might be better to do template pack expansion only where you use PathWithLaneId.

Reply:
I see :+1: 
May I just separate the files for the functions and unit tests related to `path_with_lane_id` and put them under trajectory ? This would make it easier to cut out only those related to `path_with_lane_id` when `tier4_autoware_utils` will be put into common.

Reply:
I personally think it's acceptable.